### ISSUE TITLE
[BITV]: Availability timepicker improvements

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#40699]
This is accessibility issue.

![Screenshot from 2023-09-29 09-32-23](https://github.com/nextcloud/server/assets/6078378/d23cadc7-8ff0-4a4a-bf7a-51ad59994375)

There are several improvements:

1. There are no labels and accessible labels for time. Placeholder is not a replacement for labels.

2. Every input filed have a placeholder "pick a time" which have to be `aria-label` and have to be unique for each day, like "pick a time for monday from", "pick a time for monday to"
![Screenshot from 2023-09-29 09-38-23](https://github.com/nextcloud/server/assets/6078378/d15b2a76-5a51-4ac9-ad64-834b5975bf7e)

**???** Should every next line input have a different label then previous one?
![Screenshot from 2023-09-29 09-42-21](https://github.com/nextcloud/server/assets/6078378/f7714d38-2208-402c-8106-2ba3f6433eea)

3. Dropdown menu have to be "closable" via `Esc`

Possible solution: Replace all existing timepicker with https://nextcloud-vue-components.netlify.app/#/Components/NcPickers?id=ncdatetimepickernative component. Probablu https://nextcloud-vue-components.netlify.app/#/Components/NcPickers?id=ncdatetimepickernative component have to be extended with timepicker. Please check if all points from above are covered.

**Comment by ShGKme** [comment]
> Probablu https://nextcloud-vue-components.netlify.app/#/Components/NcPickers?id=ncdatetimepickernative component have to be extended with timepicker

It supports native `type="time"`

**Comment by ShGKme** [comment]
@susnux isn't it fixed now?

**Comment by JuliaKirschenheuter** [comment]
i would say it is fixed now

**Comment by susnux** [comment]
When reopened the PR was not merged, now that it is merged I would say yes.

ref: https://github.com/nextcloud/server/pull/41339
